M2/Macaulay2/packages/AbstractSimplicialComplexes.m2

```diff
@@ -71,236 +76,194 @@ AbstractSimplicialComplex.GlobalAssignHook = globalAssignFunction
 AbstractSimplicialComplex.GlobalReleaseHook = globalReleaseFunction
 describe AbstractSimplicialComplex := K -> net expression K
 
-
 new AbstractSimplicialComplex := AbstractSimplicialComplex =>(cl) -> (
-    K := newClass(AbstractSimplicialComplex, new HashTable); -- sigh
-    K)
+     K := newClass(AbstractSimplicialComplex, new HashTable); 
+     K)
 
 spots AbstractSimplicialComplex := List => (
-  K -> sort select(keys K, i -> class i === ZZ))
+     K -> sort select(keys K, i -> class i === ZZ))
 
--- return the p-faces of a simplicial complex
+-- Return the p-faces of a simplicial complex.
 
 AbstractSimplicialComplex _ ZZ := AbstractSimplicialComplex => (K,p) -> (
-  if K#?p then K#p 
-  )
+     if K#?p then K#p
+     else {})
 
--- given a list of subsets L and A \in L decide if A is maximal
+-- Given a list of subsets L and A \in L decide if A is maximal.
 
-isMaximal :=(x,L) -> (
-myList := select(L,i -> isSubset(x,i));
-if #myList == 1 then
-return true
-else return false
-    )
+isMaximal := (x,L) -> (
+     myList := select(L,i -> isSubset(x,i));
+     if #myList == 1 then
+     return true
+     else return false)
```

Comment:
Explicitly returning `true` or `false` here is redundant -- the correct one is already the value of the expression `#myList == 1`.  You also don't need to use `return` on the last line of a function.  So this coupld be simplified to just ending with `#myList == 1`.